cmake/modules/FindOrFetchPARSEC.cmake

```diff
@@ -11,6 +11,8 @@ else (TARGET PaRSEC::parsec)
   set(CMAKE_CROSSCOMPILING OFF)
   set(CMAKE_SYSTEM_PROCESSOR ${CMAKE_HOST_SYSTEM_PROCESSOR})
   set(PARSEC_WITH_DEVEL_HEADERS ON CACHE BOOL "Install PaRSEC headers")
+  set(BUILD_TOOLS OFF CACHE BOOL "Don't build the tools by default")
+  set(BUILD_TESTING OFF)  # Don't build the tests by default
```

Comment:
probably not a good idea to set this since the standard [CTest module](https://cmake.org/cmake/help/latest/module/CTest.html) defines this and many projects (including MADNESS) use this variable also. Furthermore, if the user set `BUILD_TESTING` on CMake's command line (i.e. it's a cache variable) this definition will shadow their definition and causes all kinds of grief in debugging.

Reply:
I'd just remove this for now ... unfortunately PaRSEC will be built if the user just builds the default target ... well, they should never do `make` :)

Reply:
I thought this is restricted to the directory where defined. Anyway, I will push a fix in parsec to exclude the tests and examples from the default build if parsec is not the top project.

Reply:
Meanwhile, I have removed them from the madness script.

I have also fixed PaRSEC to define POSITION_INDEPENDENT_CODE for all libraries it creates, be they static or dynamic.
The issue is that for its python interface, tiledarray needs to define a dynamic library that includes the libparsec.a and others. As we were not compiling libparsec.a files in PIC if it was a static library, that failed.

Last, I have moved the repo we track in madness: for clarity, and to simplify multiple people pushing stuff in it, I started tracking TESSEOrg/parsec branch tesse-ttg.

The changes in parsec are in this repo/branch.